src/ModelCompiler.cmd

```diff
@@ -1,10 +1,17 @@
 @echo off
```

Comment:
I think this file became too complex. As it was, it was good for compilin Boiler # 1 and the simple events. I think we should revert and create another file called e.g. ModelCompilerNodeSet2.cmd and put that logic there.

I'm open for discussions.